### ISSUE TITLE
[Feature request] Integrate cut scenes from the designer of the CorsixTH logo

### ISSUE THREAD

**Issue by 12people** [CorsixTH/CorsixTH#1654]
#### Describe the request
9 years ago, the designer behind the CorsixTH logo proposed these amazing level change and main menu videos for CorsixTH: https://www.youtube.com/channel/UCfu1v_KfU7mnkZKsWCwm7OA.

(The original proposal was made on the forum, but as that's now defunct, the main way to contact him would likely be through YouTube.)

While adding a main menu video would probably be difficult, the level change cut scenes should be relatively simple replacements of the current cut scenes.

For those not wanting to lose the original cut scenes, a setting could be added to switch to those.

It would be a shame to lose the work that has been done on this.

**Comment by 12people** [comment]
@TheCycoONE I remember on the forum that the reason given for why these weren't implemented right away is because, to do it the right way, using modern video formats and high resolutions, it would require a new way to render videos inside of CorsixTH.

If that still holds, how about a stopgap solution, where these video files would be converted to Smacker video files, which CorsixTH currently supports?

If I figured out a way to convert the level change videos to SMK files, would you be willing to add an option to use them in the game?

**Comment by 12people** [comment]
Also, are there any other video formats that CorsixTH supports or could support easily?

**Comment by TheCycoONE** [comment]
The video format is not an issue - it should be completely doable to incorporate Walter's board game / level progression. That's been on my mind a few times lately, so thank you for linking to the youtube.

**Comment by TheCycoONE** [comment]
There is a distribution issue I haven't sorted out yet - what to do with larger binary assets like these.

**Comment by 12people** [comment]
The simplest solution IMHO would be to have a checkbox "Download and use new high-res videos where available" in two places:
1. On the initial setup screen, below the file chooser for selecting the Theme Hospital folder.
2. In the settings dialog (in case the user changes his mind later on). If the assets have been downloaded already, it should just say "Use new high-res videos where available" (without the download part).

Would that be easy to implement?

**Comment by 12people** [comment]
For reference, here is Wolter's (the author's) last forum post on the subject from Web Archive: https://web.archive.org/web/20141117203757/http://forums.corsixth.com/index.php/topic,227.75.html

He links to a Dropbox folder with source files and high-res versions, but unfortunately, the link no longer works. Would be great to request these files from him again. Source files would be great to have in case the video needs to ever be tweaked for whatever reason (e.g. new levels, new platforms, higher standard resolutions, etc.) and should ideally be available through the CorsixTH Graphics Github project.

**Comment by TheCycoONE** [comment]
Yeah, that's an option, though not an MVP requirement I think. I can add a repo for supplemental assets with it's own artifacts first, and changes to Corsix to load them.

**Comment by lewri** [comment]
I also noticed they did a nice main menu animation too. But shame it wouldn't really fit anymore with the amount of options we have.
Not easily anyway.

**Comment by 12people** [comment]
@lewri The menu in the pull-down would not work without significant changes, but the video could be repurposed as the intro screen and the menu could just be shown on top of the pull-down.
